app/src/main/res/layout/activity_scale_bar.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.mapbox.maps.MapView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/mapView"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    app:mapbox_cameraTargetLat="61.489557808394814"
+    app:mapbox_cameraTargetLng="23.735379149896005"
+    app:mapbox_cameraZoom="14.940602577615023"
+
+    app:mapbox_scaleBarBorderWidth="1dp"
+    app:mapbox_scaleBarHeight="5dp"
+    app:mapbox_scaleBarGravity="@integer/scaleBarGravity"
```

Comment:
will actually `bottom|end` work here?

Reply:
I haven't tried. Moreover, I doubt we support RTL for this view.

Reply:
Removed the separate xml values to avoid confusion.